resources/locale/en.yml

```
@@ -11,5 +11,9 @@ fof-split:
             to: Split to here
         modal:
             title: Split discussion
+            type_new_label: Split to a new discussion
+            type_existing_label: Split into existing discussion
+            search_discussion_label: Select a discussion to which to split posts
             new_discussion_label: Specify the title for the new discussion
+            split_to_discussion_help: 'Posts will be split into <b>{title}</b>'
```

```suggestion
            split_to_discussion_help: 'Posts will be split out and merged into <b>{title}</b>'
```
A little extra clarity here, I think